docs/topics/tour/kotlin-tour-classes-part-1.md

```diff
@@ -0,0 +1,279 @@
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3" id="tour-access-property-kotlin"}
+
+> To concatenate the value of a property as part of a string, you can use template expressions (`$`).
+> For example:
+> ```kotlin
+> println("Their email address is: ${contact.email}")
+> ```
+>
+{type="tip"}
+
+## Member functions
+
+In addition to declaring properties as part of an object's characteristics, you can also define an object's behavior 
+with member functions.
+
+In Kotlin, member functions must be declared within the class body. To call a member function on an instance, write the 
+function name after the instance name appended with a period `.`. For example:
+
+```kotlin
+class Contact(val id: Int, var email: String) {
+    fun printId() {
+        println(id)
+    }
+}
+
+fun main() {
+    val contact = Contact(1, "mary@gmail.com")
+    contact.printId()           //Calls member function printId()
+    //1
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3" id="tour-member-function-kotlin"}
+
+## Data classes
+
+Kotlin has **data classes** which are particularly useful for storing data. Data classes have the same functionality as 
+classes, but they come automatically with additional member functions. These member functions allow you to easily copy 
+instances of a class, compare instances, print the instance to readable output, and more. As these functions are
+automatically available, you don't have to waste time writing the same boilerplate code for each of your classes.
+
+To declare a data class, use the keyword `data`:
+```kotlin
+data class User(val name: String, val id: Int)
+```
+
+All the predefined member functions of a data class are listed in the table below:
+
+| **Function**       | **Description**                                                                                                |
+|--------------------|----------------------------------------------------------------------------------------------------------------|
+| `copy()`           | Creates a class instance by copying another, potentially with some different properties.                       |
+| `componentN()`     | Accesses properties of the class in their order of declaration. E.g. `component1` accesses the first property. |
+| `equals()` or `==` | Compares instances of a class.                                                                                 |
+| `hashCode()`       | Exposes the hash code of an instance, for comparison.                                                          |
+| `toString()`       | Prints a readable string of the class instance and its properties.                                             |
+
+See the following sections for examples of how to use each function:
+* [Copy instance](#copy-instance)
+* [Access properties](#access-data-class-properties)
+* [Compare instances](#compare-instances)
+* [Expose hash code](#expose-hash-code)
+* [Print as string](#print-as-string)
+
```

Comment:
Of all these, I'd suggest providing the following examples (in the specified order):

* Print as string
* Compare instances
* Copy instance